src/utils/settings.cpp

```diff
@@ -97,7 +101,7 @@ void check_status(const QSettings& settings, const QString& attempted_operation)
                                      : QStringLiteral("access error (consider running with an administrative role)")};
 }
 
-QString checked_get(QSettings& settings, const QString& key, const QString& fallback, std::mutex& mutex)
+QString checked_get(const QSettings& settings, const QString& key, const QString& fallback, std::mutex& mutex)
```

Comment:
> What drive-by changes do you mean?

This and the dropped newline below. I had an impression there were more, but maybe not.

> The newlines were meant to counteract the mix of new identifiers to the bucket.

Yeah didn't mean those.

Reply:
Oh, I missed this before indeed, and I agree it should be done regardless of whether this PR goes in. 

I just needed this in this PR because, as it was, `checked_get` required an lvalue ref, so line 161 wouldn't work. Const refs OTOH bind to (and extend the lifetime of) temporaries, so the const makes line 161 work. 

Anyway, I will make it a separate PR and have that as a precedence for this one.

Reply:
And the newline removal was just an oversight, I'll drop that.

Reply:
Done